build.sbt

```diff
@@ -80,6 +80,14 @@ val zioConfigDependencies = Seq(
   "dev.zio" %% "zio-config-magnolia" % zioConfigVersion
 ).map(_ excludeAll ("org.scala-lang.modules", "scala-collection-compat"))
 
+val crealyticsDependencies = Seq(
+  "com.crealytics" %% "spark-excel" % "3.4.1_0.19.0"
+).map(_.cross(CrossVersion.for3Use2_13))
+
+val poiDependencies = Seq(
+  "org.apache.poi" % "poi" % "5.2.5"
+)
+
```

Comment:
The versions need to be mentioned under TOOL VERSIONS section in sbt file.

Reply:
Added both Crealytics and PIO versions in separate section